eeps/eep-0007.md

```diff
@@ -740,7 +741,7 @@ are (probably) going to be introduced in Erlang.
 
 Thus, the high-level FFI API should be considered highly experimental
 and subject to change, depending on how type contracts will allow to
-achieve the same tasks (see [High-level API][]).  This issue will need
+achieve the same tasks (see [High-level API]).  This issue will need
```

Comment:
Did you test this? The original Markdown spec allows this link syntax: https://daringfireball.net/projects/markdown/syntax#link. It's shorthand for `[High-level API][High-level API]`. I'm not sure `[High-level API]` becomes a link.

Reply:
I've removed the link alltogether.